### PR TITLE
[IMP] *: stock b2b

### PR DESCRIPTION
**Pull request by mohammadhossam**
Enterprise PR: odoo/enterprise#70099
Task-4173210

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale_picking/views/delivery_view.xml
Comment (chevalierv) [inline addons/website_sale_picking/views/delivery_view.xml:1]: FYI `website_sale_picking`  and `onsite` has been renamed `website_sale_collect` and `in_store` in https://github.com/odoo/odoo/pull/174073

File: addons/stock_fleet/models/stock_picking_batch.py [hunk 1]
```diff
@@ -10,7 +10,8 @@ class StockPickingBatch(models.Model):
     vehicle_category_id = fields.Many2one(
         'fleet.vehicle.model.category', string="Vehicle Category",
         compute='_compute_vehicle_category_id', store=True, readonly=False)
-    dock_id = fields.Many2one('stock.location', string="Dock Location", domain="[('is_a_dock', '=', True)]")
+    dock_id = fields.Many2one('stock.location', string="Dock Location", domain="[('warehouse_id', '=', warehouse_id), ('is_a_dock', '=', True)]",
```
Comment (amoyaux) [inline addons/stock_fleet/models/stock_picking_batch.py:13]: I would rather go for something like child_of move_ids location and location_dest.
Picking type is not always valid while the move locations are

File: addons/website_sale_picking/views/delivery_view.xml
Comment (Feyensv) [inline addons/website_sale_picking/views/delivery_view.xml:1]: `website_sale_picking` is dead, long live `website_sale_collect` :eyes:

Comment (mohammadhossam) [reply addons/website_sale_picking/views/delivery_view.xml:1]: Oh, okay. Thanks for the heads up!

**Comment by amoyaux** [comment]
robodoo r+ rebase-ff

**Comment by amoyaux** [comment]
robodoo r+

**Pull request closed without merging**